src/vaadin-select.html

```diff
@@ -528,6 +559,7 @@
             }
             this.validate();
             window.removeEventListener('scroll', this._boundSetPosition, true);
+            this._wasOpened = true;
```

Comment:
This doesn't really work well. This causes a bug that the next value change after closing always triggers a `change` event (regardless of if it was user initiated or not).

```js
select.opened = true;
select.opened = false;
```